package.json

```diff
@@ -1,39 +1,41 @@
 {
-  "name": "deprem-io-backend",
-  "version": "1.0.0",
-  "description": "",
-  "main": "index.js",
-  "scripts": {
-    "start": "node index.js",
-    "dev": "npx nodemon --exec \"node index.js\"",
-    "test": "vitest run --coverage",
-    "lint": "rome check --apply-suggested .",
-    "format": "rome format --write ."
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/acikkaynak/deprem-io-backend.git"
-  },
-  "author": "",
-  "license": "ISC",
-  "bugs": {
-    "url": "https://github.com/acikkaynak/deprem-io-backend/issues"
-  },
-  "homepage": "https://github.com/acikkaynak/deprem-io-backend#readme",
-  "dependencies": {
-    "@fastify/autoload": "^5.7.1",
-    "@fastify/cors": "^8.2.0",
-    "@fastify/swagger": "^8.3.1",
-    "@fastify/swagger-ui": "^1.3.0",
-    "dotenv": "^16.0.3",
-    "fastify": "^4.13.0",
-    "mongoose": "^6.9.1",
-    "node-cache": "^5.1.2"
-  },
-  "devDependencies": {
-    "@vitest/coverage-c8": "^0.28.4",
-    "nodemon": "^2.0.20",
-    "rome": "^11.0.0",
-    "vitest": "^0.28.4"
-  }
+   "name": "deprem-io-backend",
```

Comment:
bu dosyanin indentation'ini neden degistirdik

Reply:
bunu ben de anlayamadım normalde Rome kullanmıyorum ama uyumlu olması için tüm gerekli ayarları yaptım yine de sorun oluyor

Reply:
degisiklikleri eski haline alalim lutfen